src/Uno.UI/Extensions/UIElementExtensions.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Windows.UI.Xaml;
+using Uno.Extensions;
+using Uno.Logging;
+
+namespace Uno.UI.Extensions
+{
+	public static partial class UIElementExtensions
```

Comment:
I'm not sure we should be keeping this public.

Reply:
It's in the `Uno` namespace.  But I agree it could be internal.